juan-silva/Final-Project/project/src/pages/Pokemon.js

```diff
@@ -0,0 +1,123 @@
+import { useEffect, useState } from 'react';
+import './Poke.css';
+import run from '../assets/running.gif';
+
+const url='https://pokeapi.co/api/v2/pokemon?limit=600';
+let urltemp;
+let pokList=[];
+function Pokemon(data){
+    const [pokemonList,setPokemonList]=useState([]);
+    useEffect(()=>{
+        setMyPokemons();
+    },[]);
+    const getEachPokemon=(poke)=>{
```

Comment:
@jsilva97 , Since you are not using Redux, all the logic and API consumption should be in a context